js/main.js

```diff
@@ -71,6 +115,30 @@ document.addEventListener('DOMContentLoaded', function (event) {
 });
 // Loops through data.entries array and uses renderEntry() to generate a DOM tree with content and append it to the unordered list ($uList).
 
+$uList.addEventListener('click', function (event) {
+
+  if (event.target.id !== 'pen') {
+    return;
+  }
+
+  viewSwap('entry-form');
```

Comment:
I would call this at the end of the function, so that you swap after all the inputs get filled in